unity-client/Assets/Scripts/MainScripts/DCL/Components/BaseComponent.cs

```diff
@@ -115,5 +145,7 @@ public virtual void OnPoolGet()
         }
 
         public abstract int GetClassId();
+
+        public Transform GetTransform() => transform;
```

Comment:
We should move this behaviour to `IMonoBehaviour `interface so only the components who need a game object have it. I think this is planned right?

Reply:
In fact I didn't think about this. Good advice!